pom.xml

```diff
@@ -57,7 +57,7 @@
         <enforceCorrectDependencies>true</enforceCorrectDependencies>
         <fasterxmlVersion>2.10.3</fasterxmlVersion>
         <springDataVersion>2.0.9.RELEASE</springDataVersion>
-        <caf.worker-batch-framework.version>3.4.0-SNAPSHOT</caf.worker-batch-framework.version>
+        <caf.worker-batch-framework.version>3.4.0</caf.worker-batch-framework.version>
```

Comment:
This isn't a proper version number, the component needs to be released first

Reply:
alright, I ll close the PR.thx